source/fe/fe_poly_tensor.cc

```diff
@@ -940,18 +961,22 @@ FE_PolyTensor<PolynomialType, dim, spacedim>::fill_fe_face_values(
   // on the neighborhood between two faces.
   std::fill(fe_data.sign_change.begin(), fe_data.sign_change.end(), 1.0);
 
-  if (mapping_type == mapping_raviart_thomas)
-    internal::FE_PolyTensor::get_face_sign_change_rt(cell,
-                                                     this->dofs_per_face,
-                                                     fe_data.sign_change);
+  internal::FE_PolyTensor::get_face_sign_change_rt(cell,
+                                                   *this,
+                                                   this->mapping_type,
+                                                   fe_data.sign_change);
 
-  else if (mapping_type == mapping_nedelec)
-    internal::FE_PolyTensor::get_face_sign_change_nedelec(cell,
-                                                          this->dofs_per_face,
-                                                          fe_data.sign_change);
+  internal::FE_PolyTensor::get_face_sign_change_nedelec(cell,
+                                                        *this,
+                                                        this->mapping_type,
+                                                        fe_data.sign_change);
 
   for (unsigned int i = 0; i < this->dofs_per_cell; ++i)
     {
+      MappingType mapping_type =
```

Comment:
here and below, mark this variable as `const` since it won't change in this block.

Reply:
Got it!